contracts/FeralfileArtworkV4_1.sol

```diff
@@ -0,0 +1,160 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.13;
+
+import "./FeralfileArtworkV4.sol";
+
+contract FeralfileExhibitionV4_1 is FeralfileExhibitionV4 {
+    mapping(address => uint256) public advances;
+
+    error InvalidAdvanceAddressesAndAmounts();
+    error InvalidAdvanceAddress();
+    error InvalidAdvanceAmount();
+    error InvalidSignature();
+
+    constructor(
+        string memory name_,
+        string memory symbol_,
+        bool burnable_,
+        bool bridgeable_,
+        address signer_,
+        address vault_,
+        address costReceiver_,
+        string memory contractURI_,
+        uint256[] memory seriesIds_,
+        uint256[] memory seriesMaxSupplies_
+    )
+        FeralfileExhibitionV4(
+            name_,
+            symbol_,
+            burnable_,
+            bridgeable_,
+            signer_,
+            vault_,
+            costReceiver_,
+            contractURI_,
+            seriesIds_,
+            seriesMaxSupplies_
+        )
+    {}
+
+    /// @notice set advances setting
+    /// @param addresses_ - the addresses to set advances
+    /// @param amounts_ - the amounts to set advances
+    function setAdvanceSetting(
+        address[] calldata addresses_,
+        uint256[] calldata amounts_
+    ) public onlyAuthorized {
+        if (addresses_.length != amounts_.length) {
+            revert InvalidAdvanceAddressesAndAmounts();
+        }
+        for (uint256 i = 0; i < addresses_.length; i++) {
+            if (addresses_[i] == address(0)) {
+                revert InvalidAdvanceAddress();
+            }
+            if (amounts_[i] == 0 && advances[addresses_[i]] == 0) {
+                revert InvalidAdvanceAmount();
+            }
+            if (amounts_[i] > 0) {
+                advances[addresses_[i]] = amounts_[i];
```

Comment:
This will replace the advance item if it exists yet. We could check if the advance address exists, we shouldn't allow to update the amount, we just should allow to replace the address.